GraphAlgorithms/Clique/MaxCliqueExactFinder.cs

```diff
@@ -0,0 +1,22 @@
+﻿using GraphAlgorithms.Comparers.VertexNumber;
+using GraphAlgorithms.Comparers.VertexThenEdge;
+
+namespace GraphAlgorithms.Clique
+{
+    public class MaxCliqueExactFinder : ICliqueFastFinder
```

Comment:
Maybe we could put these methods in the already existing class?

Reply:
I'd opt for the wrapper classes because otherwise we'd have to use this generic type parameter to call those methods